JudoPayDotNetTests/Headers/UserAgentTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace JudoPayDotNetTests.Headers
+{
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+
+    using JudoPayDotNet.Http;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class UserAgentTests
+    {
+        [TestCase("Mozilla / 4.0", 1)]
+        [TestCase("Mozilla / 4.0(compatible)", 0)] // The underlying .Net classes are unable to parse these values
+        [TestCase("Mozilla / 4.0(compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)", 0)] // The underlying .Net classes are unable to parse these values
```

Comment:
will do. I was trying to get a better understanding of what passes fails the user agent parser as the documentation is a little sparse. The outcome is less than optimal and from what I can see any string with any braces, semi-colons, commas etc will fail to parse. This means that we won't get any user-agent headers set by the clients most of the time. Happy to discuss this as I can't see any easy answer, and I'm not keen to write a user-agent parser.